src/search-manager/FilterByPublished.tsx

```diff
@@ -42,6 +48,25 @@ const FilterByPublished: React.FC<Record<never, never>> = () => {
   };
   const appliedFilters = publishStatusFilter.map(mode => ({ label: modeToLabel[mode] }));
 
+  const filterLabels = {
+    [PublishStatus.Published]: intl.formatMessage(messages.publishStatusPublished),
+    [PublishStatus.Modified]: intl.formatMessage(messages.publishStatusModified),
+    [PublishStatus.NeverPublished]: intl.formatMessage(messages.publishStatusNeverPublished),
+  };
+
+  const visibleFiltersToRender = visibleFilters.map((filter) => (
+    <MenuItem
+      as={Form.Checkbox}
+      value={filter}
+      onChange={() => { toggleFilterMode(filter); }}
```

Comment:
Don't you need a `key` here since you're using `map` ? Please test new things with your JS console open so you don't miss warnings.

Reply:
Updated: https://github.com/openedx/frontend-app-authoring/pull/1947/commits/df819d36c76c7d05059efb4538b470729f96cd59